packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -175,4 +177,18 @@ export default abstract class EasCommand extends Command {
     await this.analytics.flushAsync();
     return super.finally(err);
   }
+
+  protected baseErrorMessage = 'Command failed.';
+  protected baseGraphQLErrorMessage = 'GraphQL request failed.';
```

Comment:
This should be handled/caught by ESLint, but define class properties before methods.

Reply:
Well, it's not the case with other properties, like on lines 150 or 141, etc.

Reply:
Can you please fix those as well?